lib/rules/indent-function-args.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @fileoverview Check for function args proper indentation.
+ * @author uberVU
+ * @copyright 2015 uberVU. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict';
+var getNodeIndent = require('../get-node-indent.js');
+var indentSize = 4;
+
+//------------------------------------------------------------------------------
+// Rule Definition
```

Comment:
I saw that this is present in every rule from the main eslint and also in plugins like eslint-plugin-react. But I guess we can kill it :(.